README.md

```diff
@@ -201,6 +204,55 @@ Since version 1.1 of the `yoti-sdk-impl` you can set the following two system pr
 * `yoti.client.connect.timeout.ms` - the number of milliseconds that you are prepared to wait for the connection to be established. Zero is interpreted as an infinite timeout.
 * `yoti.client.read.timeout.ms` - the number of milliseconds that you are prepared to wait for data to become available to read in the response stream. Zero is interpreted as an infinite timeout.
 
+## AML Integration
+
+Yoti provides an AML check service to allow a deeper KYC process to prevent fraud. This is a chargeable service, so please contact teamgrowth@yoti.com for more information.
+
+Yoti will provide a boolean result on the following checks:
+* PEP list - Verify against Politically Exposed Persons list
+* Fraud list - Verify against  US Social Security Administration Fraud (SSN Fraud) list
+* Watch list - Verify against watch lists from the Office of Foreign Assets Control
+
+To use this functionality you must ensure:
+* Your application is assigned to your Organisation in the Yoti Dashboard - please see here for further information.
+* Within your application please ensure that you have selected the 'given names' and 'family name' attributes from the data tab. This is the minimum requirement for the AML check.
+
+The AML check uses a simplified view of the User Profile.  You need only provide the following:
+* profile.givenNames
+* profile.familyName
+* Country of residence - you will need to collect this from the user yourself
+
+To check a US citizen, you must provide two more attributes in addition to the three above:
+* Social Security Number - you will need to collect this from the user yourself
+* Postcode/Zip code
+
+### Code Example
+
+Given a YotiClient initialised with your SDK ID and KeyPair (see Client Initialisation) performing an AML check is a straightforward case of providing basic profile data.
+
+```java
+// POJOs for the data to check
+AmlAddress amlAddress = new AmlAddressBuilder()
+                            .withPostCode(“E1 6DB”)
```

Comment:
why do you use a wither syntax if it is a builder?

Reply:
Matches the style adopted in YotiClientBuilder.
It's just a fluent/readability thing.  Can change it if you wish.

Reply:
Didn't realise that. Sorry.
Then it's good to be consistent.